networks/rpc/types_test.go

```diff
@@ -42,14 +43,19 @@ func TestBlockNumberJSONUnmarshal(t *testing.T) {
 		6:  {`"0x12"`, false, BlockNumber(18)},
 		7:  {`"0x7fffffffffffffff"`, false, BlockNumber(math.MaxInt64)},
 		8:  {`"0x8000000000000000"`, true, BlockNumber(0)},
-		9:  {"0", true, BlockNumber(0)},
+		9:  {`"0"`, true, BlockNumber(0)},
```

Comment:
Can I ask why this is fixed?

Reply:
original - `9:  {"0", true, BlockNumber(0)},`
second field is `mustFail`.
This test code is decimal input fail test.
After BlockNumber cover decimal, 
this test cannot be passed (because not fail)
So I change second field `true` to `false`.
And it's moved to no.17 for test consistency. (1~16 : String hex test / 17~21 : decimal test)
now - `17:  {"0", false, BlockNumber(0)},`

`9:  {`"0"`, true, BlockNumber(0)},`
this is new test for check string decimal fail.

Reply:
Thanks for the kind explanation. It's really helpful for understanding rest of the test code :)